app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,78 @@
+package ar.com.wolox.android.example.ui.home
+
+import androidx.core.util.Pair
+import androidx.fragment.app.Fragment
+import ar.com.wolox.android.example.ui.news.NewsFragment
+import ar.com.wolox.android.example.ui.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_home.*
+import kotlinx.android.synthetic.main.fragment_viewpager.vViewPager
+import javax.inject.Inject
+import android.os.Build
+import android.view.WindowManager
+import androidx.core.content.ContextCompat
+import ar.com.wolox.android.R
+
+class HomeFragment @javax.inject.Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject
+    internal lateinit var newsFragment: NewsFragment
+    @Inject
+    internal lateinit var profileFragment: ProfileFragment
+    private lateinit var fragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    private val newsTab = 0
```

Comment:
You can move this to companion object, same as `profileTab`. You can also declare them as consts and use this notation: `NEWS_TAB`.